csharp/ql/test/format.json

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "pattern": "**/*.cs",
+    "allowMixedTabsAndSpaces": true
```

Comment:
My initial reaction is the same for the java test files; I don't think they should have mixed ws either.

Reply:
For C++ we allow mixed whitespace in test source files because we want to run our tests not only on neatly formatted input but also on ugly and realistic files.

Reply:
Right.  And I guess there can be queries of a more syntactic nature where it matters.  So it's probably fine to whitelist the tests.